### PR TITLE
New Files Added : Assignment5 & assignment6

### PR DESCRIPTION
**Pull request by Pranjali1999**

### CODE REVIEW

File: Pranjali/HTML/Assi5/assi5.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<html>
+    <head>
+        <style>
+            h1{
+                border: 2px solid black;
+                height: 500px;
+                width: 600px;
+                font-style: oblique;
+            }
+
```
Comment (akash19972306) [inline Pranjali/HTML/Assi5/assi5.html]: remove the extra lines from this whole file

File: Pranjali/HTML/Assi6/assi6.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <p>
+            <label for="fname"><b>First Name:</b></label>
+            <input type="text" id="fname" name="firstname" placeholder="First Name" />
+        </p>
+        <p class="c">
+            <label for="lname"><b>Last Name:</b></label>
+            <input type="text" id="lname" name="lastname" placeholder="Last Name" />
+        </p>
+        <p>
+            <label></label>
```
Comment (akash19972306) [inline Pranjali/HTML/Assi6/assi6.html]: why you taken label...and there is nothing in that label remove it

File: Pranjali/HTML/Assi6/assi6.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .a {
+                font-family: Arial, Helvetica, sans-serif;
+                background-color: rgb(137, 167, 224);
+                text-align: right;
+            }
+
```
Comment (akash19972306) [inline Pranjali/HTML/Assi6/assi6.html]: remove extra lines

File: Pranjali/HTML/Assi6/assi6.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                border-bottom-left-radius: 8px;
+                border-top-right-radius: 8px;
+            }
+        </style>
+    </head>
+    <body>
+        <h2 class="a" id="corners">MI-9 Intern HTML Assesment</h2>
+        <br />
+        <hr />
+        <h3 class="b" id="corners"><i>Personal Details</i></h3>
```
Comment (akash19972306) [inline Pranjali/HTML/Assi6/assi6.html]: use <em> tag instead <i> tag. beacause <i> tag is outdated. and also use <strong> tag instead of <b> tag

**Review by akash19972306 — CHANGES_REQUESTED** [review]

**Review by akash19972306 — APPROVED** [review]

**Review by vinod2406 — APPROVED** [review]
ok

**Review by vishnukopnar — APPROVED** [review]

**Review by Bhavika222 — APPROVED** [review]

**Review by Akash-ing — APPROVED** [review]

File: Pranjali/HTML/Assi6/assi6.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        </style>
+    </head>
+    <body>
+        <h2 class="a" id="corners">MI-9 Intern HTML Assesment</h2>
+        <br />
+        <hr />
+        <h3 class="b" id="corners"><em>Personal Details</em></h3>
+        <p>
+            <label for="fname"><strong>First Name:</strong></label>
+            <input type="text" id="fname" name="firstname" placeholder="First Name" />
```
Comment (SnehaMakade) [inline Pranjali/HTML/Assi6/assi6.html]: placeholder should be like First name so make this change in all input boxes

File: Pranjali/HTML/Assi6/assi6_2.html [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+        <form>
+            <h3 class="b" id="corners"><em>Personal Details</em></h3>
+            <fieldset>
+                <legend>Academic Details</legend>
+                <p>
+                    <label><strong>Skill Set:</strong></label>
+                    <input type="checkbox" id="HTML" name="skillset" value="HTML" />
+                    <label for="html">HTML</label>
+                    <input type="checkbox" id="Javascript" name="skillset" value="Javascript" />
+                    <label for="javascript"> Javascript</label>
```
Comment (SnehaMakade) [inline Pranjali/HTML/Assi6/assi6_2.html:33]: remove all extra spaces

File: Pranjali/HTML/Assi6/assign6.html [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+        </style>
+    </head>
+    <body>
+        <h2 class="a" id="corners">MI-9 Intern HTML Assesment</h2>
+        <br />
+        <hr />
+        <h3 class="b" id="corners"><i>Personal Details</i></h3>
+        <p>
+            <label for="fname"><b>First Name:</b></label>
+            <input type="text" id="fname" name="firstname" placeholder="First Name" />
```
Comment (SnehaMakade) [inline Pranjali/HTML/Assi6/assign6.html:29]: change placeholder in all input boxes

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

File: Pranjali/HTML/Assi5/assi5.html [hunk 8]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            h1{
```
Comment (SnehaMakade) [inline Pranjali/HTML/Assi5/assi5.html]: there should be space before paranthesis and class name like h1 {}, make this change everywhere

File: Pranjali/HTML/Assi6/assi6.html [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+        </style>
+    </head>
+    <body>
+        <h2 class="a" id="corners">MI-9 Intern HTML Assesment</h2>
+        <br />
+        <hr />
+        <h3 class="b" id="corners"><em>Personal Details</em></h3>
+        <p>
+            <label for="firstname"><strong>First Name:</strong></label>
+            <input type="text" id="firstname" name="firstname" placeholder="First Name" />
```
Comment (SnehaMakade) [inline Pranjali/HTML/Assi6/assi6.html]: palceholders second words 1st letter should be small like First name make this change everywhere

File: Pranjali/HTML/Assi6/assi6_2.html (hunk 6 above)
Comment (SnehaMakade) [inline Pranjali/HTML/Assi6/assi6_2.html:33]: remove all extra space

File: Pranjali/HTML/Assi6/assign6.html (hunk 7 above)
Comment (SnehaMakade) [inline Pranjali/HTML/Assi6/assign6.html:29]: not done changes

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
remove extra spaces and change placeholder names like First name in all input boxes

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**